### DISCUSSION TITLE
interval in plot + tooltip

### DISCUSSION THREAD

**Discussion by macbryla** [observablehq/plot#1903] [Q&A] [answered]
I have a simple bar chart, data is at 'day' level and I am using interval = "day/week/month" to display it by selected interval without needing to fetch remote data for each aggregation level . This works well. 

The challenge I am having is making the tooltip follow this interval. What I get is tooltip showing the 'day' level as there is no interval option, i.e. it treats the bar as a stacked bar and showing for example multiple 'day' level numbers on a 'month' level bar.

Is there a way to make this work?
Thank you for your help.

**Comment by Fil** [comment, marked as answer]
Can you share an example of the code that doesn't work? Here's what I'm getting (the tip shows the interval):
<img width="658" alt="Capture d’écran 2023-10-26 à 09 04 44" src="https://github.com/observablehq/plot/assets/7001/56215a45-00a4-489f-be1f-094fa7186b3d">

**Reply by macbryla** [reply]
Thank you Philippe
It was misunderstanding on my part, I was using interval in a wrong context.
After watching the latest YouTube videos from Allison, it all came together. Great content by the way.
